### ISSUE TITLE
Yesterdays released articles not showing

### ISSUE THREAD

**Issue by issekinichou** [brianrak/satori-reader-mobile-beta#49]
**Describe the bug**
Under Recently Released, top of the list is 08 August 2018
Akiko
Unraveled
Teenage Intern
No heading or articles for yesterday
Articles page counters not synced. All three of the series with recent releases show one number lower than website.
Tap on Koibito
Sort by date to newest first
Top of list is Unraveled.
New article not showing. Can see it on website.

**To Reproduce**
Sign in
Dashboard, Recently Released
Sync, No change.
Articles page. Counters not updated. All three of the series with recent releases show one number lower than website.
Sync, no change.
Tap Koibito. Latest article not showing

**Expected behavior**
Expected header 15 August  2018 and yesterdays articles, as visible on website
Articles page counters should match website

**Smartphone (please complete the following information):**
 iPad mini 4
iOS 11.4.1

**Additional context**
Incidentally, after writing the above, I signed out on the app and desktop browser. Then I noticed iPad browser tab. It was still showing the Review Export page you get after tapping the Complete button on the export page. Because this page does not look like satori reader, I often forget to hit the browser back button and log out.
I logged out on this browser tab and reopened the mobile app. Still showing above bug. Recent header 08 Aug.

**Comment by brianrak** [comment]
Under investigation.

**Comment by brianrak** [comment]
Fixed. This was a server-side issue, so we were able to fix it without requiring an update to the app. Please synchronize within the app and the articles should appear properly. I would appreciate it if you would also be on the watch next Wednesday at 19:00 UTC to verify that the next batch appears properly. Thank you!

**Comment by issekinichou** [comment]
Brian,

1900 UTC = 2000 BST according to an online converter.
I go out wed nights and usually avoid using screens after 8 pm most nights.
Will check Thursday morning about 0600 BST (british summer time).

This weeks recent released appeared this morning, but note that the second artilce had <em> tags in the title. That stood out clearly to me, but then I averted my eyes to avoid the preview text as it is a series I am saving for later. I think it was
<em> Shiawase to wa</em>  [Final Episode] Image was a profile face.
I added a comment on Github #47.

Richard
